soundbay/train.py

```diff
@@ -176,6 +182,7 @@ def main(args):
         scheduler_args=args.optim.scheduler,
         model_args=args.model.model,
         logger=logger,
+        num_unfrozen=args.optim.num_unfrozen,
```

Comment:
Please make sure - if running this with a regular model (no freezing), this arg will pass `None`?

Reply:
I'm afraid this implementation demands we add this argument to all optim configuration files.
I made sure it works as default 'False' even if the argument is only stated, with no attached value (in which case it passes "None" and is then turned into "False" but the framework doesn't work if the argument is none-existent in the optim file.